### PR TITLE
#269/organisms/sidebar

### PR DESCRIPTION
**Pull request by pranotoism**
organisms sidebar #269

### CODE REVIEW

File: src/Components/Organisms/FOHeader/index.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ export const FOHeader = memo((props: IFOHeader) => {
     handleLogin,
     handleRegister,
     logout,
+    search,
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOHeader/index.tsx:14]: unused vars

File: src/Components/Organisms/FOSideBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
+import React, { memo } from "react";
+import { styled, useTheme } from "@mui/material/styles";
+import makeStyles from "@mui/styles/makeStyles";
+
+import Drawer from "@mui/material/Drawer";
+import CssBaseline from "@mui/material/CssBaseline";
+import List from "@mui/material/List";
+
+import Divider from "@mui/material/Divider";
+import IconButton from "@mui/material/IconButton";
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOSideBar/index.tsx:10]: Should be use atoms component

File: src/Components/Organisms/FOSideBar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <CssBaseline />
+      <Drawer
+        className={color ? classes.color : ""}
+        data-testid={testID}
+        sx={{
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOSideBar/index.tsx:70]: All style should be available to customize

**Review by gilangdarmawan — CHANGES_REQUESTED** [review]

**Comment by gilangdarmawan** [comment]
Add `"no-unused-vars": ["error", { "vars": "all", "args": "after-used", "ignoreRestSiblings": false }]` in eslint rules to check all unused vars. I've got 6 problems when run linter using this rules.

**Comment by ardiwine** [comment]
@pranotoism please review the requested changes.

File: src/Components/Organisms/FOSideBar/index.tsx (hunk 2 above)
Comment (pranotoism) [reply src/Components/Organisms/FOSideBar/index.tsx:10]: done

File: src/Components/Organisms/FOSideBar/index.tsx (hunk 3 above)
Comment (pranotoism) [reply src/Components/Organisms/FOSideBar/index.tsx:70]: done

**Review by ardiwine — APPROVED** [review]

**Pull request merged**